tests/linux_kernel/test_stack_trace.py

```diff
@@ -97,3 +98,13 @@ def test_prog(self):
             self.prog.stack_trace(Object(self.prog, "struct pt_regs", value={})).prog,
             self.prog,
         )
+
+    def test_stack__repr_pretty_(self):
```

Comment:
We don't really need to run the `_repr_pretty_()` test with root privilege and the kernel module loaded.

Do we want to mock `StackTrace` and `StackFrame` instead? Says in `tests/test_stack_trace.py`.

Reply:
We don't have a great way to test stack traces otherwise right now, so I think it's fine to do it here for now.